.travis.yml

```diff
@@ -5,9 +5,9 @@ jdk:
 
 install:
   - cd $TRAVIS_BUILD_DIR/maven-sdk3
-  - mvn clean package
+  - mvn clean package -q
```

Comment:
I think `cd` needs to be on the same line as the `mvn` or `gradlew` commands.

Also, instead of running this all as an `install` command, you can specify a matrix of commands to execute https://docs.travis-ci.com/user/build-matrix/

Reply:
I don't completely see how this relates to `matrix`... 

Also this is exactly what users of this project would do locally, KIS?

Reply:
These are different builds that are unrelated, right?
And I thought every line would run in a separate shell

Reply:
They are indeed unrelated, but the purpose of a build matrix is more building with different environments, language version, ... ? _I do not have any experience with this 'matrix' principles, I'm just reading some documentation & try to understand how it works / how we could use it_

No, the lines are executed after each other in series.

Reply:
A build matrix allows you to run multiple builds in parallel. You can use it for anything you want.
In this case, I would probably choose to put working directory and command in an envvar in the matrix.

But in my opinion it is not worth blocking this PR for this minor comment.